test/rules/variableNameRuleTests.ts

```diff
@@ -92,4 +92,23 @@ describe("<variable-name>", () => {
             return optionallyValidFailures.some((f) => f.equals(failure));
         }));
     });
+
+    it("ensures keywords are banned optionally", () => {
```

Comment:
"ensure keywords can optionally be banned" is more in parallel with the other tests